inc/surveyanswer.class.php

```diff
@@ -173,11 +176,17 @@ static function showSurvey(CommonGLPI $item, $preview = false) {
             }
          }
          self::displayAnswer($question, $value);
-         echo "</div>";
-         echo "</div>";
+         echo "</td>";
+         echo "</tr>";
       }
 
+      echo "</tbody>";
+      echo "</table>";
       echo "</div>";
+
+      echo "<script type='text/javascript'>";
+      echo "$('td', '#mainformtable').addClass('w-50');";
+      echo "</script>";
```

Comment:
Original code did not work for formcreator UI.

```suggestion
      echo Html::scriptBlock("
         // Isolate variables in a self calling function
         (function(){
            // Set table content width
            const setTableWidth = function() {
               $('#mainformtable td').addClass('w-50');
            };

            // Throttled function to avoid spamming the function on repeated events
            const setTableWidthDebounced = _.throttle(setTableWidth, 500, false);

            // Run once immediatly
            setTableWidth();

            // Run the function on each container change, to make sure it is applied to all rows
            $('#mainformtable').on('DOMSubtreeModified', function() {
               setTableWidthDebounced();
            });
         })();
      ");
```